marconi-chain-index/src/Marconi/ChainIndex/Indexers.hs

```diff
@@ -110,11 +110,11 @@ initialCoordinator indexerCount =
 -- The points should/could provide shared access to the indexers themselves. The result
 -- is a list of points (rather than just one) since it offers more resume possibilities
 -- to the node (in the unlikely case there were some rollbacks during downtime).
-type Worker = Coordinator -> FilePath -> IO [Storable.StorablePoint ScriptTx.ScriptTxHandle]
+type Worker = Int -> Coordinator -> FilePath -> IO [Storable.StorablePoint ScriptTx.ScriptTxHandle]
```

Comment:
Can we use `Word64` as that's the type used by consensus as well?

Also, how about a newtype for this?

Reply:
I'd avoid a newtype here: worker is really a type alias, I don't see a scenario where we'd like to distinguish `Worker` from its alias  (but I'd be happy to change my mind if you have one), which is the main (unique/) purpose of a newtype.

Reply:
@koslambrou Newtype for the security parameter you mean?

Reply:
Yes :) Newtype for the security param, not the worker.

Reply:
My bad, so yes.

Reply:
21830c7d1